main/main.py

```diff
@@ -29,5 +29,9 @@
 
 if config.DEVELOPMENT:
   from werkzeug import debug
-  app.wsgi_app = debug.DebuggedApplication(app.wsgi_app, evalex=True)
+  try:
+    app.wsgi_app = debug.DebuggedApplication(
+        app.wsgi_app, evalex=True, pin_security=False)
```

Comment:
Before the #423 is there.. let's keep the current style.. comma in the end and `)`

```python
    app.wsgi_app = debug.DebuggedApplication(
        app.wsgi_app, evalex=True, pin_security=False,
      )
```